src/Sections-AboutPage/Trainers/trainers.css

```diff
@@ -0,0 +1,149 @@
+
+ .trainers-section {
+     color: #ffffff;
+     background-image: url("/Images/space-background.jpg");
+     background-size: cover;
+     background-position: center;
+     background-repeat: no-repeat;
+     position: relative;
+     /*Needed for stacking */
+     padding-bottom: 35px;
+ }
+
+ .trainers-headline {
+    text-align: center;
+    padding: 40px;
+ }
+
+ .trainers-headline h1 {
+     color: #FFE03B;
+     font-family: Roboto;
+     font-size: 30px;
+     font-style: normal;
+     font-weight: 600;
+     line-height: normal;
+     margin: 0;
+ }
+
+ .trainers-headline p {
+     color: #FFF;
+     font-family: Roboto;
+     font-size: 13px;
+     font-style: normal;
+     font-weight: 800;
+     line-height: normal;
+ }
+
+ .trainers-wrapper {
+     width: 100%;
+     height: 100%;
+     overflow: auto;
+     display: flex;
+     scroll-behavior: smooth;
+     gap: 12px;
+     position: relative;
+     /* margin-top: 35px; */
+ }
+
+ .trainer-card img {
+     height: 100%;
+ }
+
+ .trainers-wrapper::-webkit-scrollbar {
+     width: 10px;
+ }
+
+ .trainers-wrapper::-webkit-scrollbar-thumb {
+     background-color: #292929;
+     border-radius: 5px;
+ }
+
+ .trainers-wrapper::-webkit-scrollbar-thumb:hover {
+     background-color: #333;
+ }
+
+ .trainer-card {
+     position: relative;
+     text-align: center;
+     width: 324px;
+     height: 441px;
+     flex-shrink: 0;
+     border-radius: 20px;
```

Comment:
I like the rounded corners of the card! Different from the design, but like the change:)